_episodes/02-starting-with-data.md

```diff
@@ -18,7 +18,11 @@ objectives:
     - "Perform basic mathematical operations and summary statistics on data in a Pandas DataFrame."
     - "Create simple plots."
 keypoints:
-    - "FIXME"
+    - "Libraries enable you to add functions to Python." 
```

Comment:
I'm not sure we should say that libraries enable us to add functions... They allow us extend the limits of what Python can do...

Reply:
Oh, that's a good point - you could add your own functions without installing any libraries.

Reply:
```suggestion
    - "Libraries enable us to extend the functionality of Python." 
```

Reply:
Maybe that?

Reply:
👍